Assets/Tests/EditMode/NearestNeighborInterpolator2DTests.cs

```diff
@@ -0,0 +1,79 @@
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+using UnityEngine.TestTools;
+using UnityEngine.TestTools.Utils;
+
+public class NearestNeighborInterpolator2DTests {
+  private const float _epsilon = 1e-3f;
+
+  [Test]
+  public void Interpolate_InvalidPoints_ReturnsEmptyResult() {
+    LogAssert.ignoreFailingMessages = true;
```

Comment:
I believe this is a global static flag, so once a test sets it to true, all other tests afterwards will silently swallow their errors.

[LogAssert.Expect](https://docs.unity3d.com/Packages/com.unity.test-framework@1.1/api/UnityEngine.TestTools.LogAssert.html#UnityEngine_TestTools_LogAssert_Expect_UnityEngine_LogType_System_String_) would be better here.